scripts/run_ansible_test

```diff
@@ -48,10 +48,9 @@ cat  "${HOME}/code/ansible_collections/cifmw/general/tests/sanity/ignore.txt" >>
     "${HOME}/code/ansible_collections/cifmw/general/tests/sanity/ignore-${ansible_version}.txt"
 
 pushd ${HOME}/code/ansible_collections/cifmw/general
-${ansible_test} sanity --test validate-modules
 if [ -d tests/integration ]; then
     ${ansible_test} integration --color=yes
 fi
```

Comment:
so no longer running the sanity test?

Reply:
Ouch, restored, my fault with my local, cause I removed it locally to skip them